### PR TITLE
Increased power output update scenario

### PR DESCRIPTION
**Pull request by radujipa**
This is the update strategy for increasing power output on USBs
for Pi2 boards. The main change was made in kano-settings repo.

In addition to: https://github.com/KanoComputing/kano-settings/pull/447

### CODE REVIEW

**Comment by radujipa** [comment]
[As a CK2 Pro user, I want devices that use lots of power to work when plugged into my USB hub](https://trello.com/c/cwrMzGuI/1081-as-a-ck2-pro-user-i-want-devices-that-use-lots-of-power-to-work-when-plugged-into-my-usb-hub)

File: kano_updater/scenarios.py [hunk 1]
```diff
@@ -734,4 +711,51 @@ def beta_3100_to_beta_3101(self):
         pass
 
     def beta_3101_to_beta_3102(self):
-        pass
+        try:
+            from textwrap import dedent
```
Comment (skarbat) [inline kano_updater/scenarios.py]: Had I knew about `dedent` in the past! Great one!

File: kano_updater/scenarios.py [hunk 2]
```diff
@@ -734,4 +711,51 @@ def beta_3100_to_beta_3101(self):
...
+
+            except ImportError:
+                pass
+
+            with open(tmp_path, 'a') as tmp_config:
+                tmp_config.write(extra_config)
+
+            if use_transactions:
+                _trans().copy_from(tmp_path)
+
```
Comment (skarbat) [inline kano_updater/scenarios.py]: minor.. I would remove empty line for clarity

Comment (radujipa) [inline kano_updater/scenarios.py]: Fixed 👍

**Pull request merged**